### DISCUSSION TITLE
433mhz on Sonoff T3 model

### DISCUSSION THREAD

**Discussion by bkbartk** [arendst/Tasmota#14702] [Q&A] [answered]
Hello,

I bought a sonoff t3 model, the T3 TX 2CH to be precise. And I noticed, next to wifi this one also has 433mhz. My question is, how can I use it? Can I receive and transmit data? and if not should I turn it off to be on the safe side?

**Comment by ascillato** [comment, marked as answer]
From the manufacturer website: https://itead.cc/product/sonoff-tx-series-wifi-smart-wall-switches/?sscid=21k6_1sp96& it states that the 433 MHz module is RECEIVER ONLY. If you have a 433 key tag or a Sonoff Bridge 433, you can send a turn on / turn off command. The 433 RF receiver module is not controlled by Tasmota. You should have a learn button to configure that module to receive a command from RF. If you don't configure it, nothing should happen. The RF module ONLY simulates a push to the buttons.

Tasmota template for this device: https://templates.blakadder.com/sonoff_TX-T3EU2C.html
